fs.go

```diff
@@ -935,25 +977,29 @@ func (h *fsHandler) handleRequest(ctx *RequestCtx) {
 	statusCode := StatusOK
 	contentLength := ff.contentLength
 	if h.acceptByteRange {
-		hdr.setNonSpecial(strAcceptRanges, strBytes)
+		hdr.SetCanonical(strAcceptRanges, strBytes)
```

Comment:
If `fsSmallFileReader` does not support `ByteRange`, cannot set `Accept-Ranges: bytes`.